charts/application/values.yaml

```diff
@@ -382,3 +382,9 @@ initDefaults:
       cpu: 500m
       memory: 100Mi
   restartPolicy: Always
+
+# List of tolerations, will be taken over like-for-like to pod-spec
+tolerations: []
+
+# Affinity object, will be taken over like-for-like as well
+affinity: {}
```

Comment:
is there some example for this object? or link to some docu?

Reply:
https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/